scripts/dev/e2e.py

```diff
@@ -69,18 +71,6 @@ def _prepare_test_environment(config_file: str) -> None:
     )
```

Comment:
There are two things I would like to improve (if wanted at all) in either this or a separate PR:
- changing the print statements so that they accurately reflect that a cluster role binding and a cluster role instead of a role binding and role are created (makes it easier to spot it right away in future debugging proceses)
- fixing the comments of existing functions: some function names have changed and the comments do not reflect that. The comment for _prepare_test_environment could also be more specific.

Reply:
feel free to do them in the same PR if they are relatively small changes.